server/db/helper.js

```diff
@@ -3,8 +3,41 @@ const {
   Concept,
   Answer,
   User,
+  UserBadge,
+  Badge,
 } = require('./index');
 
+const insertUserBadge = (userId, badgeId) => UserBadge.create({
+  idUser: userId,
+  idBadge: badgeId,
+});
+
+const updateUserXp = (userId, badgeId) => {
+  return Badge.findOne({
+    where: {
+      id: badgeId,
+    },
+    attributes: ['experiencePoints'],
+  })
+    .then((xp) => {
+      User.increment({
+        totalExperiencePoints: xp.dataValues.experiencePoints,
+      }, {
+        where: { id: userId },
+      });
+    })
+    .catch((err) => {
+      console.error(err);
+    });
+};
+
+const getUserBadge = (userId, badgeId) => UserBadge.findOne({
```

Comment:
Can you remove the getUserBadge helper since Brandon is working on a detailed version of that.